server/src/app.ts

```diff
@@ -1,12 +1,78 @@
 import express, { Application, Request, Response } from "express";
+import { v4 as uuidv4 } from "uuid";
+
+interface Item {
+  id: string;
+  title: string;
+  description: string;
+  dueDate?: number;
+  createdAt: number;
+}
 
 const app: Application = express();
 
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 
-app.get("/", (req: Request, res: Response) => {
-  res.send("Hello, Express!");
+app.get("/alive", (req: Request, res: Response) => {
+  res.send("Hello, World!").status(200);
+});
+
+// DATABASE SIMULATION
+
+const items: Item[] = [];
```

Comment:
please use `Map` class for storage. Storing everything in a simple array is a viable option, but it makes the cost of looking up one item O(n), whereas with a Map it's O(1).

There are examples for the map usage in the ticket.